common/app/views/fragments/atoms/storyquestions.scala.html

```diff
@@ -0,0 +1,26 @@
+@(storyquestions: model.content.StoryQuestionsAtom, isAmp: Boolean)(implicit request: RequestHeader)
+
+@if(!isAmp) {
+    <div class="submeta user__question">
+    <span class="js-storyquestion-atom-id" style="display: none" id="@storyquestions.id"></span>
+
+    <h1 class="user__question-title">Finished this article but want to know more? <span class="user__question-title--secondary">Let us know what questions you want answered.</span></h1>
+
+    @for(questions <- storyquestions.data.editorialQuestions) {
+        @for(qs <- questions) {
+            <p>
+                <div class="user__question-section">
+                  <h3>@qs.questionSetTitle</h3>
+                </div>
+                @for(question <- qs.questions) {
+                    <p class="user__questions-text--wrapper">
+                        <span id="js-question-text-@question.questionId" class="user__question-text"><span>@question.questionText</span></span>
+                        <a href="#" id="@question.questionId" class="user__question-upvote js-ask-question-link">Ask this <span class="tick">@fragments.inlineSvg("tick", "icon")</span></a>
```

Comment:
Nit: for readability, could you have what is inside this link on a separate line. What about: 

```
<a href="#" id="@question.questionId" class="user__question-upvote js-ask-question-link">
    Ask this 
    @fragments.inlineSvg("tick", "icon")
</a>
```

Notice I removed the span as well because  the icon will already have a span with the class `tick` why next two of them?